tests/test_modules.py

```diff
@@ -16,13 +16,11 @@
 
 from pytest_chalice.handlers import RequestHandler
 
-from chalicelib.models import ModuleModel
-
 
 def test_list_versions_response_schema(client: RequestHandler) -> None:
     fqmn = "namespace/name/provider"
 
-    response = client.get(f"/v1/{fqmn}/versions")
```

Comment:
The "/v1" mount point should be dynamically discovered using the discovery endpoint